### DISCUSSION TITLE
How to know time of  executeQuery and ResultSet.next

### DISCUSSION THREAD

**Discussion by dowonderatwill** [glowroot/glowroot#856] [General]
In Glowroot, plugins->jdbcPlugin->ResultSet Navigation, if this check box is checked then the time of executeQuery+ResultSet.next is shown. If we uncheck it then we see the time of executeQuery, as it will not track the ResultSet.next navigation. Here, I was in need that I wanted to see the time for both executeQuery and ResultSet.next. Because setFetchSize missing etc could be known if we know both the time. Generally to confirm I run the application once with that checkbox checked and another run with the check box unchecked. 

Was wondering whether there is any other better way for this in Glowroot?

**Comment by trask** [comment]
hi @dowonderatwill, I think you could configure your own Timer instrumentation for Result.next under Configuration > Instrumentation
